### PR TITLE
Update partner programmes

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Prepare "Partners" section for release

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit http://0.0.0.0:8002/ and click on `Partners` in the navigation
- Navigate to http://0.0.0.0:8002/partners and see the available partner programmes. QA the links on this page
- QA all the links on the following pages: http://0.0.0.0:8002/partners/public-cloud, http://0.0.0.0:8002/partners/ihv-and-oem, http://0.0.0.0:8002/partners/desktop, http://0.0.0.0:8002/partners/channel-and-reseller, http://0.0.0.0:8002/partners/gsi & http://0.0.0.0:8002/partners/become-a-partner

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2559 & https://github.com/canonical-web-and-design/canonical.com/issues/214

## Screenshots

[if relevant, include a screenshot]

### CODE REVIEW

**Comment by waynecrosby** [comment]
@solazio Looking good, just a few comments:
1. /partners/become-a-partner - Remove full stop from last bullet
2. On all logo clouds with a link, can you remove the '›' before the external link icon, there are also a couple on links at the bottom of the /partners/ihv-and-oem page too
3. We have a new External link icon, links below:
https://assets.ubuntu.com/v1/88bc1143-external-link-dark.svg
https://assets.ubuntu.com/v1/1c837fa5-external-link-blue.svg
https://assets.ubuntu.com/v1/6d60d9bc-external-link-white.svg
https://assets.ubuntu.com/v1/7a72740c-external-link.svg
4. Sticky Nav: I think that keeping the main nav sticky is a bit strange when you are in the Partner section, I think it would be better to keep the last most relevant level sticky, so in this case the white nav

File: templates/partners/gsi.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  </div>
+  <div class="u-fixed-width">
+    <ul class="p-inline-images u-no-margin--bottom">
+      {% for partner in partners %}
+      <li class="p-inline-images__item">
+        <img class="p-inline-images__logo" src="{{ partner.logo }}" alt="{{ partner.slug}}">
+      </li>
+      {% endfor %}
+    </ul>
+    <p class="u-align-text--center"><a href="https://partners.ubuntu.com/find-a-partner" class="p-link--external">See all System Integrators partners&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/partners/gsi.html:36]: This one has both external and chevron.

**Comment by anthonydillon** [comment]
Adding design -1 as there Wayne left comments.

**Comment by waynecrosby** [comment]
@solazio The only thing that doesn't look right is the external link, it looks too small and when you zoom in is still the old one. Please use the new set below, thanks:

https://assets.ubuntu.com/v1/907418dd-external-link-blue-lrg.svg
https://assets.ubuntu.com/v1/368a3004-external-link-dark-lrg.svg
https://assets.ubuntu.com/v1/a1bf55df-external-link-lrg.svg
https://assets.ubuntu.com/v1/d511e989-external-link-white-lrg.svg

**Comment by solazio** [comment]
@waynecrosby as discussed this will be implemented in vanilla at some point in the future. For now I suggest we use the old icons.

**Comment by waynecrosby** [comment]
@solazio Agreed. +1

**Comment by lyubomir-popov** [comment]
I was asked to comment on the subnav, I think the spacing is a bit too tight both horizontaly and vertically. A comparison with ubuntu.com:



Taking into consideration that the subnav wording is more verbose (longer words, sometimes more than one  - like "Global system integrators") proper horizontal spacing is even more important.

**Comment by spencerbygraves** [comment]
I would agree with @lyubomir-popov here. Can it be consistent with u.com?

**Comment by solazio** [comment]
@deadlight see comments above if you're the one working on this. If not, @SirSamTumless highlight the comments to whoever will work on it.

File: templates/partners/desktop.html [hunk 2]
```diff
@@ -83,30 +89,14 @@ <h3 class="p-matrix__title">The key to corporate fleet</h3>
...
-        </li>
-        <li class="p-inline-images__item">
-          <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/df4150e2-NEC-logo.svg" alt="NEC">
+          <img class="p-inline-images__logo" src="{{ partner.logo }}" alt="{{ partner.slug}}">
         </li>
+        {% endfor %}
       </ul>
       <p class="u-align-text--center">
-        <a href="/partners/find-a-partner?category=desktop">See all Desktop partners&nbsp;&rsaquo;</a>
+        <a href="https://partners.ubuntu.com/find-a-partner" class="p-link--external">See all Desktop partners</a>
```
Comment (tbille) [inline templates/partners/desktop.html:99]: Can you update this link? to point to `/partners/...`

File: templates/partners/gsi.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  </div>
+  <div class="u-fixed-width">
+    <ul class="p-inline-images u-no-margin--bottom">
+      {% for partner in partners %}
+      <li class="p-inline-images__item">
+        <img class="p-inline-images__logo" src="{{ partner.logo }}" alt="{{ partner.slug}}">
+      </li>
+      {% endfor %}
+    </ul>
+    <p class="u-align-text--center"><a href="https://partners.ubuntu.com/find-a-partner" class="p-link--external">See all System Integrators partners</a></p>
```
Comment (tbille) [inline templates/partners/gsi.html:36]: Can you update this link? to point to `/partners/...`

File: templates/partners/ihv-and-oem.html [hunk 4]
```diff
@@ -62,7 +62,7 @@ <h4 class="p-muted-heading u-no-max-width">A selection of our Hardware partners<
       </li>
     {% endfor %}
     </ul>
-    <p class="u-align-text--center"><a href="/partners/find-a-partner?filters=hardware">See all Hardware partners&nbsp;&rsaquo;</a></p>
+    <p class="u-align-text--center"><a href="https://partners.ubuntu.com/find-a-partner" class="p-link--external">See all IHV/OEM partners</a></p>
```
Comment (tbille) [inline templates/partners/ihv-and-oem.html:65]: Can you update this link? to point to `/partners/...`

File: templates/partners/public-cloud.html [hunk 5]
```diff
@@ -46,32 +46,13 @@ <h4 class="p-muted-heading u-no-max-width">A selection of our cloud partners</h4
...
-      </li>
+      {% for partner in partners %}
       <li class="p-inline-images__item">
-        <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/d4d3cd69-Joyent+logo.svg" alt="Joyent">
+        <img class="p-inline-images__logo" src="{{ partner.logo }}" alt="{{ partner.slug}}">
       </li>
+      {% endfor %}
     </ul>
-    <p class="u-align-text--center"><a href="/partners/find-a-partner?filters=certified-public-cloud">See all Public Cloud partners&nbsp;&rsaquo;</a></p>
+    <p class="u-align-text--center"><a href="https://partners.ubuntu.com/find-a-partner" class="p-link--external">See all Public Cloud partners</a></p>
```
Comment (tbille) [inline templates/partners/public-cloud.html:55]: Can you update this link? to point to `/partners/...`

File: templates/partners/ihv-and-oem.html [hunk 6]
```diff
@@ -144,7 +144,7 @@ <h2>Co-sell with Canonical through the partner portal</h2>
   <div class="row">
     <div class="col-6">
       <p>Open source technology is growing in demand. We want you to benefit from this unique opportunity, and take advantage of the investment Canonical is making in its partner community. We are committed to giving you the tools and programs to succeed, from discounts and financial incentives, to sales and technical support teams.</p>
-      <p><a href="/partners">Find out more about our partners&nbsp;&rsaquo;</a></p>
+      <p><a href="https://partners.ubuntu.com/" class="p-link--external">Find out more about our partners</a></p>
```
Comment (tbille) [inline templates/partners/ihv-and-oem.html:147]: Can you update this link? to point to `/partners/...`

**Comment by nottrobin** [comment]
Superseded by @pmahnke's branch here: https://github.com/canonical-web-and-design/canonical.com/pull/230

**Pull request closed without merging**